source/routing/raptor.cpp

```diff
@@ -402,9 +434,16 @@ RAPTOR::Journeys RAPTOR::compute_all_journeys(const map_stop_point_duration& dep
                                               bool clockwise,
                                               const boost::optional<navitia::time_duration>& direct_path_dur,
                                               const size_t max_extra_second_pass) {
+    log4cplus::Logger logger = log4cplus::Logger::getInstance(LOG4CPLUS_TEXT("logger"));
```

Comment:
Would be interesting to create a new logger (`"raptor"`? maybe) not to pollute the current one and be able to turn it off if it spits out too many things

Reply:
Yes, done ! https://github.com/CanalTP/navitia/pull/2937/commits/53cd072b2a5f6596daa177532ad871f67da15f12